docs/source/index.rst

```diff
@@ -58,7 +58,11 @@ as a native runtime to achieve improvement in terms of query efficiency and quer
    Comet Plugin Overview <contributor-guide/plugin_overview>
    Development Guide <contributor-guide/development>
    Debugging Guide <contributor-guide/debugging>
+<<<<<<< HEAD
```

Comment:
Looks like this is a merge/rebase error

Reply:
Ah, thanks! [1c37b49](https://github.com/apache/datafusion-comet/pull/422/commits/1c37b490b5e0d80d6de5b3ff6844f489a87777cf)